### PR TITLE
feat(core): mode in query string

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7904** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7904?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/workbench/entities/view.ts [hunk 1]
```diff
@@ -34,7 +36,7 @@ export class View extends Entity<{
     const activeTabId = get(this._activeSidebarTabId$);
     const tabs = get(this.sidebarTabs$);
     return tabs.length > 0
-      ? (tabs.find(tab => tab.id === activeTabId) ?? tabs[0])
+      ? tabs.find(tab => tab.id === activeTabId) ?? tabs[0]
```
Comment (pengx17) [inline packages/frontend/core/src/modules/workbench/entities/view.ts]: is this safe?

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/modules/workbench/entities/view.ts (hunk 1 above)
Comment (EYHN) [reply packages/frontend/core/src/modules/workbench/entities/view.ts]: bad prettier

Comment (pengx17) [reply packages/frontend/core/src/modules/workbench/entities/view.ts]: need to dedupe prettier in affine repo.

**Pull request merged**